<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkDialog</class>
 <widget class="QWidget" name="NetworkDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>kNet Network</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Hostname:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="machineIP">
       <property name="text">
        <string>Machine IP text.</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Connections:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QTreeWidget" name="connectionsTree">
       <column>
        <property name="text">
         <string>Item</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string># of threads:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="numRunningThreads">
       <property name="text">
        <string># threads text.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
